src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs, selectedTabId, onTabSelected,
+}) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title, content }) => (
+            <li
+              key={id}
+              className={classNames({ 'is-active': id === selectedTab.id })}
```

Comment:
you check that current tab is selected several times in your code
let's create separate function for this check or just separate boolean variable inside the map callback and will use this variable where we need it